### PR TITLE
organization details page

### PR DESCRIPTION
**Pull request by lebeza**
@shashank-naik (
enable conditional field module
Go to admin/structure/conditional_fields/node/organization
target field: field_other_treatment_program_of
control field: field_treatment_program_offering
The target field is: visible 
when the control field: has value
Click on add dependency and make sure the next page looks like the picture below
image

Add Organization Page Summary 
admin/structure/types/manage/organization/fields/add-field
Field: Text (plain), 
label: Organization Page Summary 

Other Treatment Program Offerings fields
admin/structure/types/manage/organization/fields/add-field
Field: Text (plain),
label: Other Treatment Program Offerings

### CODE REVIEW

File: web/themes/custom/digital_toolkit/src/styles/page/_organization.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  .tobacco_treatment {
+    @include full-width;
+    background: $primary-white;
+    padding: 32px 0;
+    .tobacco_treatment_programs {
+      .treatments {
+        background: $accent-light-grey;
+        padding: 17px 22px;
+        margin: 0 22px 0 0;
+        border-top: 6px solid #1c5e86;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/page/_organization.scss]: `border-top: 6px solid $primary-blue;`

File: web/themes/custom/digital_toolkit/src/styles/page/_organization.scss [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      }
+      .org_tobacco_treatment {
+        padding: 0;
+        .title {
+          font-size: 24px;
+          color: #1c5e86;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/page/_organization.scss]: `color: $primary-blue;`

File: web/themes/custom/digital_toolkit/src/styles/page/_organization.scss [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+      .treatment-program:nth-last-child(2) {
+        display: none;
+      }
+      .org_entity {
+          li {
+            list-style: none;
+            &::before {
+              content: "\2022";
+              color: #1c5e86;
```
Comment (wansooseo) [inline web/themes/custom/digital_toolkit/src/styles/page/_organization.scss]: `color: $primary-blue;`

**Comment by wansooseo** [comment]
@lebeza Left a couple of comments. Please update `#1C5E86` to `$primary-blue`

**Comment by lebeza** [comment]
@wansooseo  fixed the primary blue colors

**Pull request merged**